Runtime/Scripts/Room.cs

```diff
@@ -58,7 +59,22 @@ public ConnectInstruction Connect(string url, string token)
             request.Connect = connect;
 
             Utils.Debug("Connect....");
-            var resp = FfiClient.SendRequest(request);
+            var resp = await FfiClient.SendRequest(request);
+            Utils.Debug($"Connect response.... {resp}");
+            return new ConnectInstruction(resp.Connect.AsyncId, this);
```

Comment:
We should get rid of or revise `ConnectInstruction` as it is a `YieldInstruction` and we would like to disconnect from Coroutines.
We just should be able to call it like `await Connect(...)` and receive any return data (or nothing).  Currently it returns a coroutine that should be again "awaited"